libs/styles/typography.css

```diff
@@ -57,7 +57,7 @@
 }
 
 .con-button {
-  color: #FFF;
+  color: var(--color-white);
```

Comment:
nit: But in my mind this seems very similar to `#FFF`. If you end up changing the value to anything other than white it won't make any sense.

Reply:
Agreed. Sometimes the css vars can be overly robust